modules/openapi-generator/src/main/resources/dart2/serialization/native/native_class.mustache

```diff
@@ -2,73 +2,73 @@ class {{{classname}}} {
 {{>dart_constructor}}
 {{#vars}}
   {{#description}}
-  /// {{{.}}}
+    /// {{{.}}}
   {{/description}}
   {{^isEnum}}
     {{#minimum}}
       {{#description}}
-  ///
+        ///
```

Comment:
are all the whitespace changes necessary?

Reply:
That was unintentional, I've updated the PR